src/app/legacy/containers/CpsRecommendations/ErrorBoundary.tsx

```diff
@@ -0,0 +1,31 @@
+import { useContext } from 'react';
+import nodeLogger from '../../../lib/logger.node';
+import { RECOMMENDATIONS_MISSING_DATA } from '../../../lib/logger.const';
+import { RequestContext } from '../../../contexts/RequestContext';
+
+const logger = nodeLogger(__filename);
+
+const checkRecommendations = recommendations => {
+  const isValidRecommendations = recommendations.every(
+    recommendation => recommendation.headlines && recommendation.locators,
```

Comment:
How much of the recommendations should we check? I'm thinking not too much as it would be a bit of a pain to keep updated.

Reply:
I'd agree with this, given a truly generic solution doesn't seem possible better to keep it fairly simple